internal/cli/atlas/dbusers/create.go

```diff
@@ -131,7 +139,7 @@ func (opts *CreateOpts) validate() error {
 		return errors.New("missing role for the user")
 	}
 
-	if opts.isExternal() && opts.password != "" {
+	if opts.isExternal() && (opts.password != "") {
```

Comment:
[q/np] Do we need the extra brackets here?

Reply:
thanks @colm-quinn. Addressed in latest.